hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/snapshot/OMSnapshotPurgeResponse.java

```diff
@@ -172,7 +172,7 @@ private void cleanupSnapshotChain(OmMetadataManagerImpl metadataManager,
             nextPathSnapInfo.getTableKey(), nextPathSnapInfo);
       } else if (nextGlobalSnapInfo != null) {
         nextGlobalSnapInfo.setGlobalPreviousSnapshotId(
-            snapInfo.getPathPreviousSnapshotId());
+            snapInfo.getGlobalPreviousSnapshotId());
```

Comment:
We might also have to change in Line 170 as well.

Reply:
looks like copy/paste bug :)